webapi/src/main/resources/application.conf

```diff
@@ -274,8 +274,8 @@ app {
     }
 
     triplestore {
-        dbtype = "fuseki"
-        // dbtype = "graphdb"
+        // dbtype = "fuseki"
```

Comment:
yes, I wondered about this, too :-)

@subotic ok, just this small thing and then you could merge it back.